### DISCUSSION TITLE
Checkbox nightmare

### DISCUSSION THREAD

**Discussion by AuracleTech** [community/community#49917] [Accessibility]
### Select Topic Area

Product Feedback

### Body

![image](https://user-images.githubusercontent.com/40531871/224704019-4987aa1b-4ce1-485e-9e4c-c6d8955e8b75.png)
Please change the colors of the checked checkboxes to have more contrast
It's fine if you leave it grayscaled but please make it something other than light gray on light gray

**Comment by AuracleTech** [comment]
![image](https://user-images.githubusercontent.com/40531871/224707054-b0658092-d438-4289-a6ff-d823b26cfca6.png)
This is already much better for example

**Comment by cehfisher** [comment]
Hello @AuracleTech - thank you for taking the time to provide this feedback! Our Accessibility team is actively working on this issue. We will follow-up here with any additional questions or updates. Thanks again! (cc @queenofcorgis)

**Reply by AuracleTech** [reply]
Accidentally opened the same issue twice, must really show how much that simple issue affects me
Oh this is a discussion, well that explains it

**Reply by cehfisher** [reply]
No worries @AuracleTech - thank you for taking the time to provide this feedback! The good news is, our Accessibility team is actively working on this issue. We will follow-up here with any additional questions or updates. Thanks again!
